### ISSUE TITLE
PAPI Placeholders don't work

### ISSUE THREAD

**Issue by haeiven** [DiscordSRV/DiscordSRV#792]
Since few devs builds from 1.19.0 to 1.19.1, the PAPI integration doesn't works.

```
[21:18:40] [Server thread/ERROR]: [PlaceholderAPI] Failed to init placeholder expansion from class: com.discordsrv.placeholderapi.DSRVExpansion
[21:18:40] [Server thread/ERROR]: [PlaceholderAPI] github/scarsz/discordsrv/dependencies/google/common/collect/ImmutableMap
```

**Comment by Vankka** [comment]
We fixed this in development builds by including the expansion right in the plugin, https://snapshot.discordsrv.com

**Comment by Vankka** [comment]
Fixed in release 1.20.0
